keyboards/atset/at2/readme.md

```diff
@@ -0,0 +1,25 @@
+# ATSET AT2
+
+A special 2 key keyboard powered by QMK.
+
+* Keyboard Maintainer: [ATSET](https://github.com/anubhavd7)
+* Hardware Supported: RP2040
+* Hardware Availability: Interest Check
```

Comment:
Coud you please fill this out properly? I could not find an IC thread on Geekhack for this board. An image would also be nice.